### PR TITLE
Adds newMsgTransfer and changes voteType to enum (number)

### PR DESCRIPTION
**Pull request by thomasmost**
⚡ [Asana Task](https://app.asana.com/0/1189045082777889/1201474845033045/f)

What Changes
---
* Adds a `Message` type
* Changes VoteType to enum (with explicit number assignments) [See Cosmos VoteOption](https://github.com/allinbits/cosmos-sdk/blob/69ab58ed2e7f50f4a5d2726454a1c467f17f4e09/x/gov/types/gov.pb.go#L59-L64)
* Adds newMsgTransfer and test

Questions
---
* Since we plan on setting expiry with a timestamp, rather than a block height, should our SDK be "opinionated" about that? E.g. should we only accept a timestamp, and always set the block height to undefined (or null)?
* Do we have opinions about versioning this library? This is a breaking change, so bumping the major version seems like the obvious way forward, but it's a little ugly that it's out of sync with the thing it's versioning (which is our Kava chain)—I suppose we could go to v9? (since this upgrade will be the launch of Kava 9) **I think I like this**

### CODE REVIEW

File: src/msg/cosmos/index.ts [hunk 1]
```diff
@@ -59,8 +59,48 @@ function newMsgVoteGovernance(
...
+// 	return &MsgTransfer{
+// 		SourcePort:       sourcePort,
+// 		SourceChannel:    sourceChannel,
+// 		Token:            token,
+// 		Sender:           sender,
+// 		Receiver:         receiver,
+// 		TimeoutHeight:    timeoutHeight,
+// 		TimeoutTimestamp: timeoutTimestamp,
+// 	}
+// }
```
Comment (downing034) [inline src/msg/cosmos/index.ts:76]: do we still need this?

**Review by downing034 — APPROVED** [review]

**Review by stephensprinkle — COMMENTED** [review]
Looks good!

**Review by stephensprinkle — APPROVED** [review]

File: src/msg/cosmos/index.ts (hunk 1 above)
Comment (thomasmost) [reply src/msg/cosmos/index.ts:76]: nope good catch

**Comment by stephensprinkle** [comment]
Re: your Q's for Feedback:

1. IMO, only the timestamp is necessary since timeoutHeight is unused (IIRC).
2. I noticed the version bump and was going to ask.  I personally like to keep module versions decoupled, but I'm not strongly opinionated.

**Comment by thomasmost** [comment]
> I noticed the version bump and was going to ask. I personally like to keep module versions decoupled, but I'm not strongly opinionated. I do like the idea of the versions matching for clarity around what they're currently supporting, but in saying that, do we ever foresee a point where there is a breaking change in the SDK w/in support of a specific chain version?

Going forward I don't think we'd ever **want** to publish a breaking change to our SDK that didn't correspond to a new (breaking) version of Kava... we'd just have to keep ourselves honest about it when deprecating/renaming methods

**Pull request merged**